test/jdk/javax/swing/text/html/StyleSheet/bug8257665.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * <Copyright>
+ */
+
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.function.Consumer;
+import javax.swing.JEditorPane;
+import javax.swing.SwingUtilities;
+import javax.swing.text.GlyphView;
+import javax.swing.text.View;
+
+/*
+ * @test
+ * @key headless
```

Comment:
No need to specify headless tag. If it's not @key headful, then it is implicitly headless

Reply:
I believe the test is _headful_ because it uses SwingUtilities.invokeAndWait even though it does not show any UI.

Reply:
I've used `SwingUtilities.invokeAndWait` just to be on the safe side that "everything Swing should be executed on the EDT" but I guess I could drop it as no UI ever shown?

Reply:
I am not sure that if we use invokeAndWait(), it needs to be headful..There are many test where we use invokeAndWait but they are headless. But it will be good to see what the test is doing visually so you probably can make it headful.

Reply:
Please, see if the option I've added to save an image capture of the editor component appears sufficient for the visual inspection.